plugins/tracer.py

```diff
@@ -64,50 +67,57 @@ def transaction(self):
                          self.base.transaction.install_set])
         erased = set([package.name for package in
                       self.base.transaction.remove_set])
+        packages = [Package(p, time.time()) for p in list(installed | erased)]
 
-        args = ["tracer", "-n"] + list(installed | erased)
-        process = subprocess.Popen(
-            args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        out, err = process.communicate()
-        _print_output(out, err)
+        try:
+            tracer = TracerFacade(packages)
+            tracer.render()
 
-        if len(out) != 0:
-            print("\n" + _("For more information run:"))
-            print("    sudo tracer -iat " + str(self.timestamp))
+            if len(tracer.apps) != 0:
+                print("\n" + _("For more information run:"))
```

Comment:
> this belongs to topic above, sorry for messing up.

So, lets move the conversation here @ignatenkobrain :-)

> Actually I would like to see tracer output immediately. Users are lazy and wants output immediately rather than execute command.

For sure users are lazy and I completely agree with all your quote ^^. However now you can see the output immediately like it always was. That's because of

    view = DefaultView()
    ...
    return view.render() 

That command suggested for getting more information is to run tracer in [interactive](http://docs.tracer-package.com/en/latest/user-guide/#interactive-mode) mode and browse those applications that DNF plugin already printed.